### DISCUSSION TITLE
I don't know HTML and CSS, can I use tauri ?

### DISCUSSION THREAD

**Discussion by monkeycc** [tauri-apps/tauri#9414] [General]
Can you provide a GUI framework and components that can be directly used ?

**Comment by FabianLars** [comment]
We're not opposed to the idea of having a built-in component library or something but it's not something we will work on anytime soon (if ever). There are tons of existing js component libraries out there, i assume one of them works for you.

If i misunderstood the question, and you don't know JS either, then Tauri may simply not be what you're looking for. You can use js-less frontend frameworks like yew or leptos (if we're talking about rust) but there are still web apis involved.

If you're looking for a gui framework without web tech, then maybe https://iced.rs/ (also Rust) is something for you?

**Comment by monkeycc** [comment]
I do want to have a built-in component library
Because there are too many JS frameworks, it is difficult to learn

Thank you for your answer
A very comprehensive answer
It's helpful to me
